<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="user_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>961</width>
     <height>221</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>username</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operator_id</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>level</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>delete</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/trash.svg</normaloff>../icons/trash.svg</iconset>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Change</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>170</red>
      <green>255</green>
      <blue>255</blue>
     </color>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/wrench-adjustable-circle-fill.svg</normaloff>../icons/wrench-adjustable-circle-fill.svg</iconset>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>32</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>User Manager
Check your users, add, delete or modify users.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_user">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background:rgba(50, 255, 50,125)}
QPushButton:hover{background:rgba(50, 255, 50,255)}</string>
   </property>
   <property name="text">
    <string>add new user</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/plus.svg</normaloff>../icons/plus.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
